### ISSUE TITLE
AlphaDropout layer seems broken

### ISSUE THREAD

**Issue by randomsock** [tensorflow/tfjs#1617]
To get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

#### TensorFlow.js version
1.1.2

#### Browser version
Node.js 10.15.3

#### Describe the problem or feature request
Using `tf.layers.alphaDropout` breaks with (example dims):

```
Error in fused matMul: inner shapes (100) and (32) of Tensors with shapes 32,100 and 32,100 and transposeA=false and transposeB=false must match.
```

Using plain `tf.layers.dropout` works as expected.

#### Code to reproduce the bug / link to feature request
Adding AlphaDropout to one of the tutorials:

```
#!/bin/env node

const tf = require('@tensorflow/tfjs');
require('@tensorflow/tfjs-node');

const model = tf.sequential();
model.add(tf.layers.dense({units: 100, activation: 'relu', inputShape: [10]}));
model.add(tf.layers.alphaDropout({rate: 0.5}));
model.add(tf.layers.dense({units: 100, activation: 'relu'}));
model.add(tf.layers.dense({units: 1, activation: 'linear'}));
model.compile({optimizer: 'sgd', loss: 'meanSquaredError'});

const xs = tf.randomNormal([100, 10]);
const ys = tf.randomNormal([100, 1]);

model.fit(xs, ys, {
    epochs: 100
});
```

Output:

```
2019-05-27 15:28:12.002115: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
Epoch 1 / 100
(node:3636) UnhandledPromiseRejectionWarning: Error: Error in fused matMul: inner shapes (100) and (32) of Tensors with shapes 32,100 and 32,100 and transposeA=false and transposeB=false must match.
    at Object.assert (...\@tensorflow\tfjs-core\dist\util.js:105:15)
    at matMul_ (...\@tensorflow\tfjs-core\dist\ops\fused_ops.js:67:10)
    at Object.matMul (...\@tensorflow\tfjs-core\dist\ops\operation.js:46:29)
    at Object.dot (...\@tensorflow\tfjs-layers\dist\backend\tfjs_backend.js:367:26)
    at droppedInputs (...\@tensorflow\tfjs-layers\dist\layers\noise.js:214:31)
    at Object.inTrainPhase (...\@tensorflow\tfjs-layers\dist\backend\tfjs_backend.js:666:23)
    at ...\@tensorflow\tfjs-layers\dist\layers\noise.js:217:26
    at ...\@tensorflow\tfjs-core\dist\engine.js:328:22
    at Engine.scopedRun (...\@tensorflow\tfjs-core\dist\engine.js:338:23)
    at Engine.tidy (...\@tensorflow\tfjs-core\dist\engine.js:327:21)
```
